root/static/scripts/edit/URLCleanup.js

```diff
@@ -2636,30 +2636,6 @@ const CLEANUPS: CleanupEntries = {
       url = url.replace(/^https:\/\/mainlynorfolk\.info\/([^/]+)(?:\/index\.html)?$/, 'https://mainlynorfolk.info/$1/');
       return url;
     },
-    select: function (url, sourceType) {
-      const m = /^https:\/\/mainlynorfolk\.info\/(?:[^/]+)\/(records|songs)?/.exec(url);
-      if (m) {
-        const prefix = m[1];
-        switch (prefix) {
-          case 'songs':
-            if (sourceType === 'work') {
-              return [LINK_TYPES.otherdatabases.work, LINK_TYPES.lyrics.work];
-            }
-            break;
-          case 'records':
-            if (sourceType === 'release') {
-              return LINK_TYPES.otherdatabases.release;
-            }
-            break;
-          default: // artist
-            if (sourceType === 'artist') {
-              return LINK_TYPES.otherdatabases.artist;
-            }
-            break;
-        }
-      }
-      return false;
-    },
```

Comment:
Your commit https://github.com/metabrainz/musicbrainz-server/pull/2184/commits/4fed82e7fe908cf87902361c1ed05543004f4d6f removes auto-selecting “other databases” too. From comments https://github.com/metabrainz/musicbrainz-server/pull/2184#issuecomment-901833627 and https://github.com/metabrainz/musicbrainz-server/pull/2184#issuecomment-901888214 it should just not auto-select “lyrics” instead.

When adding `https://mainlynorfolk.info/folk/songs/acrossthewesternocean.html` to http://localhost:5000/work/aaaab3b7-b743-4bf4-978c-bb3c02cf9913/edit no type is currently selected, while it seems “other databases” should be auto-selected and “lyrics” should be selectable.

Reply:
https://github.com/metabrainz/musicbrainz-server/pull/2184#issuecomment-891475945 suggests neither of the two to be autoselected if there's more than one - if that's the easiest implementation and it would need heavy refactoring to have "always" vs "optional", then I'd rather keep it as all optional. After all, most cases (bandcamp for example) can be one of the possible options, several at the same time, or rarely none of them even.

Reply:
Auto-selecting only one relationship type is implemented already, “select” is for “always” and “types” for “allowed” if I recall correctly.

Reply:
The current logic is whether we use auto-selection and block user from adding other relationships, or we don't auto-select at all, as discussed before.

Reply:
Implemented in commit <https://github.com/metabrainz/musicbrainz-server/pull/2184/commits/448ea12d62103fb5fb808d600f33143b6e7f0650>, thanks.

Note: The above linked discussion mentioned both approaches, and auto-selecting multiple relationship types was also part of the final roadmap. Not having selected any type in this case would have require editors to manually select the mandatory relationship type each time. Now editors just have to select the optional relationship type if needed.